generate/generate.js

```diff
@@ -38,23 +26,27 @@ const generate = async () => {
 
   let bytecodes = [];
   const { bytecode: token } = require(`../build/contracts/Token.json`);
-  bytecodes.push(['Token', address(PREDEPLOY_ADDRESS_START, 0), token]);
+  bytecodes.push(['Token', ethers.utils.getAddress('0x0000000000000000000000000000000000000800'), token]);
+
+  // add NFT bytecodes
+  const { bytecode: nft } = require(`../build/contracts/NFT.json`);
+  bytecodes.push(['NFT', ethers.utils.getAddress('0x0000000000000000000000000000000000000801'), nft]);
```

Comment:
NFT is not fully implemented yet, and changing the address will break the mandala testnet. 
NFT should use `0x0000000000000000000000000000000000000805`

Reply:
it's been for a while like that